app/services/user_event_creator.rb

```diff
@@ -57,6 +65,10 @@ def create_event_for_existing_device(event_type:, user:, device:, disavowal_toke
     )
   end
 
+  def disavowal_token
```

Comment:
can we call this `build_disavowal_token` since it's not memoized?

Reply:
Got this in 17fbbc1042d45f733e1b0eb8ce068f12ffd3be38